src/static/css/nav-secondary.less

```diff
@@ -9,6 +9,10 @@
 @nav-secondary_link__disabled-outline:         @gray-20;
 @nav-secondary_link__disabled-border:          @nav-secondary_link__disabled-bg;
 
+// TODO: Refactor these variable to use existing CF font size variables.
+@font-size-small: 16px;
+@font-size-large: 18px;
```

Comment:
Yeah agreed. These are not variables I created, but I found them tucked away further down in the file, so I just brought them to the top with the other variables. They should be scrutinized in another PR. They're in use in `unit(@font-size-small / @base-font-size-px, em);`, for example.